.github/workflows/build_and_test.yml

```diff
@@ -614,3 +614,106 @@ jobs:
           docker tag rocketchat/${{ matrix.service }}-service:${IMAGE_TAG} rocketchat/${{ matrix.service }}-service:${RELEASE}
           docker push rocketchat/${{ matrix.service }}-service:${RELEASE}
         fi
+
+  release_snap_amd64:
+    # Obs: tracks can’t be opened dynamically. We have to explicitly open them by request when a new major version gets released, otherwise this task will fail.
+    env:
+      SNAPCRAFT_REPO: RocketChat/rocketchat-server-snap
+      TESTS_REPO: RocketChat/rocketchat-release-tests
+    runs-on: ubuntu-18.04
+    needs: deploy
+    if: github.event_name == 'release'
+    steps:
+      - name: Parse environment variables
+        run: |
+          export ROCKETCHAT_VERSION=${{ github.event.release.tag_name }}
+          echo "SNAP_TRACK=${ROCKETCHAT_VERSION:0:1}.x" >> $GITHUB_ENV
+          echo "ROCKETCHAT_VERSION=$ROCKETCHAT_VERSION" >> $GITHUB_ENV
+          [[ -z "$(echo ${{ env.ROCKETCHAT_VERSION }} | grep rc)" ]] && echo "CANDIDATE=false" >> $GITHUB_ENV
+
+      - name: Clone snapcraft repo
+        if: env.CANDIDATE == 'false'
```

Comment:
```suggestion
```
plan is to make more mindful releases, i.e. fill candidate track with cadidate releases, edge with edge, and so on.